src/Component/Cart.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import Product from './Product';
+
+function Cart(props) {
+  const { cartList } = props;
+  const getTotal = () => {
+    let total = 0;
+    cartList.forEach((product) => {
+      total += product.price;
+    });
+    return total;
+  };
+  return (
+    <div>
+      <div className="Cart">
+        <h1>
+          Cart:
+          {' '}
+          $
+          {getTotal()}
+        </h1>
+        {cartList.map((product) => (
+          <Product
+            key={product.image}
```

Comment:
@prabhatVinsol use any other attribute for key

Reply:
Done